inc/vcf/string_constants.hpp

```diff
@@ -198,6 +199,15 @@ namespace ebi
     const std::string ZIP = ".zip";
     const std::string ZLIB = ".z";
 
+    // Network protocols
+    const std::string HTTP = "http://";
+    const std::string HTTPS = "https://";
+    const std::string FTP = "ftp://";
+    const std::string FTPS = "ftps://";
+
+    // ENA REST API for remote fasta access
```

Comment:
We recently found out about a new ENA API that retrieves the requested version of a contig, instead of just the latest one. Everyone in the EVA team should know which is it, could you please ask and replace it? Otherwise I think this is ready to merge.